Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'puma',                 '~> 4.3.5', '>= 4.3.5'
 gem 'pundit',               '~> 2.1'
 gem 'rack-cors',            '>= 1.1.1', '~> 1.1'
 gem 'rails',                '~> 5.2.2'
+gem 'sidekiq',              '~> 5'
```

Comment:
it uses version 6.2.1, at least minor version should be specified with `~>`

Reply:
I made it match our rails version - I will update it to `~> 5.2`, we can update it to 6 when we upgrade our rails version.